jib-gradle-plugin/src/main/java/com/google/cloud/tools/jib/gradle/JibExtension.java

```diff
@@ -56,6 +56,11 @@
  *       '/path/on/container/file2': 123
  *     ]
  *   }
+ *   outputPaths {
+ *     tar = file('jib-image.tar')
+ *     digest = file('jib-image.digest')
+ *     imageId = file('jib-image.id')
```

Comment:
I don't like this example much, since it will put these files at the project root, different from the current default behavior. It can create confusion to those new to read our code.